src/routes/docs/references/+page.markdoc

```diff
@@ -45,10 +45,13 @@ Securely store files with advanced compression, encryption and image transformat
 {% cards_item href="/docs/references/cloud/client-web/functions" title="Functions" %}
 Deploy and scale serverless functions in secure, isolated runtimes.
 {% /cards_item %}
+{% cards_item href="/docs/references/cloud/server-nodejs/messaging" title="Messaging" %}
+Send push notifications, text, or emails to users or groups of users using your app.
```

Comment:
Let's clarify what text is:

```suggestion
Send push notifications, SMS, or emails to users or groups of users using your app.
```

Reply:
Sure 👍

I picked this up here: https://github.com/appwrite/website/blob/main/src/routes/docs/products/messaging/%2Bpage.markdoc#L4
Should we change it here too?